table/options/options.go

```diff
@@ -608,33 +623,76 @@ type (
 	}
 )
 
+// WithAddColumn adds column in AlterTable request
+//
+// Deprecated: use AddColumn instead
 func WithAddColumn(name string, typ types.Type) AlterTableOption {
 	return column{
 		name: name,
 		typ:  typ,
 	}
 }
 
+// AddColumn adds column in AlterTable request
+func AddColumn(name string, typ types.Type) AlterTableOption {
+	return column{
+		name: name,
+		typ:  typ,
+	}
+}
+
+// WithAlterAttribute changes attribute in AlterTable request
+//
+// Deprecated: use AlterAttribute instead
```

Comment:
Even if deprecate old names - replace it by call right method.
It will prevent sync functional in the future and prompt right usage within code.